src/hooks/useWalletConnection.ts

```diff
@@ -131,21 +137,29 @@ export const useWalletConnection = () => {
 
       const walletConnection = getWalletConnection({ walletType });
 
+      console.log(selectedWalletType);
+
       try {
         if (!walletConnection) {
           throw new Error('Onboarding: No wallet connection found.');
         } else if (walletConnection.type === WalletConnectionType.Email) {
         } else if (walletConnection.type === WalletConnectionType.OAuth) {
           if (!isConnectedWagmi && ready && !authenticated) {
-            const provider = wallets[walletType].oAuthProvider;
-            if (provider) {
-              try {
-                await initOAuth({
-                  provider,
-                });
-              } catch (error) {
-                setSelectedWalletError(error.message);
-                log('useWalletConnection/connectWallet', error);
+            if (attemptedOAuth) {
+              dispatch(setAttemptedOAuth(false));
```

Comment:
this doesn't quite work because i need to actually reset this outside of the connectWallet function. need to look into this more